### PR TITLE
Added Gluster command to fetch volume info

### PR DESCRIPTION
**Pull request by chawlanikhil24**
Added command to fetch info over gluster volumes

### CODE REVIEW

File: uploader.v2.json [hunk 1]
```diff
@@ -615,6 +615,11 @@
             "pattern": [],
             "symbolic_name": "ovs_vsctl_show"
         },
+        {
+            "command": "/usr/sbin/gluster vol info",
```
Comment (xiangce) [inline uploader.v2.json]: Better use unify commands with its definition in insights-core

File: uploader.v2.json [hunk 2]
```diff
@@ -615,6 +615,11 @@
...
+            "pattern": [],
+            "symbolic_name": "gluster vol info"
```
Comment (xiangce) [inline uploader.v2.json]: the `symbolic_name` should be the same to the variable name in 'default.py' of insights-core, here, it should be `gluster_v_info`

File: uploader.json [hunk 3]
```diff
@@ -736,6 +736,11 @@
             "pattern": [],
             "symbolic_name": "rpm__V_packages"
         },
+        {
+            "command": "/usr/sbin/gluster volume info",
+            "pattern": [],
+            "symbolic_name": "gluster_v_info"
+        },
```
Comment (lhuett) [inline uploader.json]: please move the gluster_v_info defs between these two defs. We need to try to keep this in alphabetical order.
```
        {
            "command": "/usr/sbin/getsebool -a",
            "pattern": [],
            "symbolic_name": "getsebool"
        },
        {
            "command": "/bin/ls -l /boot/grub/grub.conf",
            "pattern": [],
            "symbolic_name": "grub1_config_perms"
        },
```

File: uploader.v2.json [hunk 4]
```diff
@@ -615,6 +615,11 @@
             "pattern": [],
             "symbolic_name": "ovs_vsctl_show"
         },
+        {
+            "command": "/usr/sbin/gluster volume info",
+            "pattern": [],
+            "symbolic_name": "gluster_v_info"
+        },
```
Comment (lhuett) [inline uploader.v2.json]: Please see comment in uploader.json

**Comment by lhuett** [comment]
This needs to be in the uploader_json branch not the master branch.

**Pull request merged**